packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -82,8 +83,8 @@ class InspectorPreferencesController extends DisposableController
       _customPubRootDirectories;
   ValueListenable<bool> get isRefreshingCustomPubRootDirectories =>
       _customPubRootDirectoriesAreBusy;
-  InspectorService get inspectorService =>
-      serviceManager.inspectorService as InspectorService;
+  InspectorService? get _inspectorService =>
+      serviceManager.inspectorService as InspectorService?;
```

Comment:
do we still need the cast?

Reply:
I've tried without it and it doesn't work because serviceManage.inspectorService is of type `InspectorServiceBase`